### DISCUSSION TITLE
Express Checkout for headless

### DISCUSSION THREAD

**Discussion by apalumbo** [Shopify/storefront-api-feedback#52] [Ideas & Requests]
Hi, given the big effort for publishing hydrogen, when do you plan to support express checkouts for headless implementations?

Thank you

**Comment by morganmccunn** [comment]
Hey @apalumbo ! We're hoping to enable this within the first quarter, though this may change as we are working through some of the prioritization right now. However, know that it is something we want to enable soon! 

There is, however, a [buy with shop pay](https://shopify.dev/custom-storefronts/tools/web-components#buy-with-shop-pay-component) component that exists today for express checkout 🙂
